rholang/src/main/scala/coop/rchain/rholang/interpreter/rholangADT.scala

```diff
@@ -110,15 +160,25 @@ case class Send(chan: Channel, data: List[Par], persistent: Boolean)
 case class Receive(binds: List[(List[Channel], Channel)],
                    body: Par,
                    persistent: Boolean,
-                   count: Int)
+                   count: Int) {
+  override def toString: String =
+    (for { (xs, chan) <- binds; if binds.nonEmpty } yield {
+      xs.mkString(", ") + " <- " + chan.toString
```

Comment:
Case in point why the pretty printing should have been a separate PR:

if persistent is true, all the <-'s should be <='s